style/elixir/README.md

```diff
@@ -0,0 +1,11 @@
+Elixir
+======
+
+[Sample](sample.ex)
+
+* Avoid macros.
+* Don't separate multiple declarations of the same function with a newline.
+* Indent pipes by two spaces when using the match operator (`=`).
+* Don't indent pipes when not using the match operator (`=`).
```

Comment:
I can't stand this practice. I don't understand why it's a thing. IMO, we should prefer consistency over (uninforced) semantic whitespace and just indent pipes 2 spaces all the time.